### PR TITLE
10634: Yellow Ribbon drop-downs not behaving correctly

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
On a school profile page, one of the calculator questions is "Will you be a Yellow Ribbon recipient?". If the user selects Yes, several conditional questions are displayed including drop-downs for "Degree Level" and "Division or school". There should be additional options in those drop-downs.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10634)

## Testing done
Testing passes locally

## Screenshots
image

## Acceptance criteria
- [x] Degree Level works as designed. Move value "Enter an amount" to be the second value in the pick list.
Note: Could possibly benefit from a re-design or better labelling.
- [x] Division or School - when value is a single value, it should behave as a disabled text box.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/sass/gi.scss [hunk 1]
```diff
@@ -1,15 +1,15 @@
-@import '~@department-of-veterans-affairs/formation/sass/shared-variables';
-@import '~@department-of-veterans-affairs/formation/sass/modules/m-modal';
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```
Comment (zurbergram) [inline src/applications/gi/sass/gi.scss:1]: looks like linting change?

Comment (elder-joseph) [reply src/applications/gi/sass/gi.scss:1]: Yes, everything other than the addition of the `disabled` class in `gi.scss` was due to linting.

Comment (shawkey-daniel) [reply src/applications/gi/sass/gi.scss:1]: I think the single quote is preferred.

Comment (elder-joseph) [reply src/applications/gi/sass/gi.scss]: Pushed a change that uses the single quote instead for the imports.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

**Pull request merged**